data/change-streams/change-streams-pre_and_post_images.json

```diff
@@ -1,827 +1,20 @@
 {
-  "description": "change-streams-pre_and_post_images",
```

Comment:
Why are we removing most of this file? It looks like the remaining piece is not valid JSON, and the `pre_and_post_images` tests don't have anything to do with CSFLE as far as I know.